### PR TITLE
Fix admin can't create record

### LINKED ISSUE #25: It is not possible to create record as Admin
Image

### PR DESCRIPTION
**Pull request by kostobog**
Fix #25
Solution:
1. on records page admin clicks create record
2. the create/record page is displayed where admin can select the form template
3. after form template is selected the form template field is disabled and the selected form is loaded from the backend
4. The loader component is shown after the selection of the form template and removed after the form is loaded from the backend.

### CODE REVIEW

File: js/components/record/TypeaheadAnswer.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ const TypeaheadAnswer = (props) => {
             isSearchable={true}
             isLoading={isLoading}
             isClearable={true}
-            isDisabled={isLoading}
+            isDisabled={isLoading || (props.isDisabled === undefined ? false : props.isDisabled) }
```
Comment (blcham) [inline js/components/record/TypeaheadAnswer.js:61]:
```suggestion
            isDisabled={isLoading || (props.isDisabled === true) }
```

Comment (blcham) [reply js/components/record/TypeaheadAnswer.js:61]: Moreover I believe it should be mentioned in `TypeaheadAnswer.propTypes`. Right @LaChope ?

**Review by blcham — CHANGES_REQUESTED** [review]
See my comment.

File: js/components/record/TypeaheadAnswer.js (hunk 1 above)
Comment (kostobog) [reply js/components/record/TypeaheadAnswer.js]: @blcham 
Resolved in commit [514bc80](https://github.com/kbss-cvut/record-manager-ui/pull/33/commits/514bc807bc185ba34f80b73b1bcb08d2e244092f)

**Review by blcham — APPROVED** [review]

**Pull request merged**